### DISCUSSION TITLE
Who is your favorite Baseball player?

### DISCUSSION THREAD

**Discussion by Ryota-keda** [NAIST-SE/AttractingContributors#1141] [2024 Attraction Contributions]
Please help me.
If you contribute to my project, I will contribute back.
https://github.com/Ryota-keda/Baseball

**Comment by thapakorn604** [comment]
Thanks for you contribution!

I also added one player to your list, please approve my request! -> https://github.com/Ryota-keda/Baseball/pull/1

**Reply by Ryota-keda** [reply]
Thank you so much!
I just approve it!

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by Ryota-keda** [reply]
Thank you!
I just contribute to yours!

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite player.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Reply by Ryota-keda** [reply]
I just contributed to your project!
Thank you!

**Reply by 3207-Rhims** [reply]
https://github.com/3207-Rhims/AttractingContributors-Movie/           , can you please contribute in my repo ? Thank you
